dbt_style_guide.md

```diff
@@ -181,48 +301,50 @@ select * from final
 
 ```
 
-- Your join should list the "left" table first (i.e. the table you are selecting `from`):
-```sql
-select
-    trips.*,
-    drivers.rating as driver_rating,
-    riders.rating as rider_rating
+## YAML style guide
 
-from trips
-left join users as drivers
-    on trips.driver_id = drivers.user_id
-left join users as riders
-    on trips.rider_id = riders.user_id
+- Every subdirectory contains their own `.yml` file(s) which contain configurations for the models within the subdirectory.
 
-```
+- A separate `.yml` file is created per top-level configuration (`sources`, `models`) that applies to the models within the subdirectory. 
 
-## YAML style guide
+- YAML files should be prefixed with an underscore to keep it at the top of the subdirectory.  
+  Example: `_sources.yml`, `_models.yml`

```

Comment:
Should we update these to the naming conventions discussed above? _sources_stripe.yml, _models_staging__stripe.yml

Reply:
Done! [Commit](https://github.com/dbt-labs/corp/pull/63/commits/a2a5e6dde8dd44258c29d38947fe847c7b93a2b6)